app/views/content/docs/bids.md

```diff
@@ -0,0 +1,31 @@
+[Documentation](/docs) &gt; Resources &gt; Bids
+
+## Bids API
+
+An bid is a theoretical bid created while processing a [bidder position](/docs/bidder_positions) (following that Auction's rules regarding increments, reserve pricing and so on).
```

Comment:
#typo `"A bid ..."`

I'm not sure where the right place is, but we could link to @dblock's blog post about english auctions that explains this modeling http://artsy.github.io/blog/2014/04/17/building-an-english-auction-with-mongodb/